assets/terraform-modules/platforms/vmware/controllers/main.tf

```diff
@@ -1,6 +1,12 @@
 locals {
   nodes_mask = split("/", var.hosts_cidr)[1]
   nodes_gw   = cidrhost(var.hosts_cidr, 1)
+
+  clc_storage = templatefile("${path.module}/clc_storage.tmpl", {
+    dns_provider : var.dns_provider,
+    etcd_fqdn : { for i, d in var.nodes_ips : format("%s-etcd%d.%s.", var.cluster_name, i, var.dns_zone) => d }
```

Comment:
@invidian these two are the entries we push into the `/etc/hosts` of the controller in case the DNS provider is set to `manual`